examples/drift_detection/README.md

```diff
@@ -138,13 +138,3 @@ In order to clean up, delete the remaining ZenML references.
 ```shell
 rm -rf zenml_examples
 ```
-
-## SuperQuick `drift_detection` run
```

Comment:
what's the reasoning behind removing this section? Does the command not work with the old evidently?

Reply:
It wont work, because it'll check out the tag 0.5.7 and that will have an unpinned evidently version in the bash script.